src/test/java/net/snowflake/client/config/SFClientConfigParserTest.java

```diff
@@ -40,6 +42,22 @@ public void testloadSFClientConfigValidPath() {
     }
   }
 
+  @Test
+  public void testloadSFClientConfigValidPathWithUnknownProperties() {
+    Path configFilePath = Paths.get("config.json");
+    try {
+      Files.write(configFilePath, CONFIG_JSON_WITH_UNKNOWN_PROPS.getBytes());
+      SFClientConfig actualConfig =
+          SFClientConfigParser.loadSFClientConfig(configFilePath.toString());
+      assertEquals("info", actualConfig.getCommonProps().getLogLevel());
+      assertEquals("/jdbc.log", actualConfig.getCommonProps().getLogPath());
+
+      Files.delete(configFilePath);
+    } catch (IOException e) {
+      fail("testloadSFClientConfigValidPath failed");
```

Comment:
we don't need catch and fail - let's throw the exception and junit will report it
especially that with fail we are loosing exception message and stacktrace

Reply:
Removed the catch for the tests which does not require it.